news/2019-10-12-beatmap-pack-management-update-#01.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: post
+title: "Beatmap Pack Management: Update #1"
+date: 2019-10-13 15:30:00 +0000
+---
+
+An update of the Beatmap Pack Management project, including new pack releases!
+
+![](/wiki/shared/news/banners/beatmap-pack-management.jpg)
```

Comment:
should use beatmap-packs.jpg and remove these duplicate images we've added in the branches that got merged together

Reply:
done.